res/values-it/derpquest_strings.xml

```diff
@@ -0,0 +1,796 @@
+
+    <!-- QS blur -->
+    <string name="blur_quicksettings">Effetto sfocato Quick Settings (beta)</string>
+    <string name="blur_quicksettings_subtitle">applica un effetto sfocato dietro i quicksettings</string>
+    <string name="qs_blur_alpha_title">Alpha sfocatura</string>
+    <string name="qs_blur_alpha_summary">Imposta la transparenza della sfocatura</string>
+    <string name="qs_blur_intensity_title">Intensità sfocatura</string>
+    <string name="qs_blur_intensity_summary">Imposta il raggio/intensità della sfocatura</string>
+
+    <!-- Buttons - Enable navbar -->
+    <string name="disable_navkeys_title">Abilita barra di navigazione</string>
+    <string name="disable_navkeys_summary">Abilita barra di navigazione a schermo acceso</string>
+
+    <!-- Status bar - icons -->
+    <string name="status_bar_icons_title">Icone barra di stato</string>
+    <string name="status_bar_system_icons_title">Icone di sistema</string>
+    <string name="status_bar_system_icons_summary">Scegli quali icone compariranno sulla barra di stato</string>
+
+    <!-- Statusbar Clock -->
+    <string name="status_bar_clock_title">Impostazioni orologio</string>
+    <string name="status_bar_clock_show">Mostra data e ora</string>
+    <string name="status_bar_clock_style_title">Stile orologio</string>
+    <string name="status_bar_clock_style_right">Destra</string>
+    <string name="status_bar_clock_style_center">Centro</string>
+    <string name="status_bar_clock_style_left">Sinistra</string>
+    <string name="status_bar_am_pm_title">AM/PM</string>
+    <string name="status_bar_am_pm_info">Ora a 24 ore</string>
+    <string name="status_bar_am_pm_normal">Normale</string>
+    <string name="status_bar_am_pm_small">Piccolo</string>
+    <string name="status_bar_am_pm_hidden">Nascosto (predefinito)</string>
+    <string name="clock_seconds_display">Mostra secondi</string>
+    <string name="clock_date_display">Data</string>
+    <string name="clock_date_display_summary">Mostra data personalizzata a sinistra dell'ora</string>
+    <string name="clock_date_display_none">Non mostrare</string>
+    <string name="clock_date_display_small">Dimensione font piccola</string>
+    <string name="clock_date_display_normal">Dimensione font media</string>
+    <string name="clock_date_style">Stile data</string>
+    <string name="clock_date_style_normal">Normale</string>
+    <string name="clock_date_style_lowercase">Minuscolo</string>
+    <string name="clock_date_style_uppercase">Maiuscolo</string>
+    <string name="clock_date_format_custom">Formato java personalizzato</string>
+    <string name="clock_date_format">Formato data</string>
+    <string name="clock_date_string_edittext_title">Deve essere necessariamente in DateFormat es. MM/dd/yy</string>
+    <string name="clock_date_string_edittext_summary">Inserisci stringa</string>
+    <string name="clock_date_right">A destra dell'ora</string>

```

Comment:
you get the idea. just find and add `\` before any `'`. Amend and force push to the same branch please

Reply:
Sure, I'll fix it asap and sorry for the inconveniences